src/domains/chart/utils/transformDataAttributes.ts

```diff
@@ -540,10 +543,54 @@ export const getAttributesStatic = (node: Element): Attributes => mapObjIndexed(
   getAttributesMap(),
 ) as Attributes // need to override because of broken Ramda typings
 
+const mapDefaultAggrMethod = (unit: string): string => {
+  const avgUnits = [
```

Comment:
Let's put this outside of the function itself (as a constant) and transform it to an object:
```
{
   percentage: true,
// or
  percent: "percent",
```
That way the complexity will be O1, since this:
```
avgUnits.some((avgUnit) => unit.includes(avgUnit)
```
will be just 
```
!!avgUnits[unit]
```

Reply:
Done